### PR TITLE
[WIP] Fix test_splitter

### PR DESCRIPTION
**Pull request by otsuman**
{{ pytest: cfme/tests/webui/test_splitter.py }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit d8de72ec0cac86194bf937f416dbcf2c243f633c:

`cfme/tests/webui/test_splitter.py`:
- :bangbang: Line 25:20: **E231** *missing whitespace after ','*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/webui/test_splitter.py [hunk 1]
```diff
@@ -66,7 +63,7 @@ def test_pull_splitter_persistence(request, appliance, model_object, destination
     navigate_to(appliance.server, 'Dashboard')
     try:
         navigate_to(model_object, destination)
-    except (TypeError, CannotScrollException):
+    except (TypeError, CannotScrollException, Exception):
```
Comment (otsuman) [inline cfme/tests/webui/test_splitter.py]: sometimes it rises `Exception: Could not open accordion VMs & Templates`

File: cfme/services/workloads.py [hunk 2]
```diff
@@ -118,10 +118,20 @@ def __init__(self, appliance=None):
         Navigatable.__init__(self, appliance)
 
 
-@navigator.register(Server)
+class Workloads(Navigatable):
```
Comment (quarckster) [inline cfme/services/workloads.py]: **QUESTION**
@psav I'm not sure is it valid or not?

File: cfme/tests/webui/test_splitter.py [hunk 3]
```diff
@@ -58,6 +53,8 @@ def test_pull_splitter_persistence(request, appliance, model_object, destination
         model_object = appliance.server
     elif issubclass(model_object, BaseCollection):
         model_object = model_object(appliance)
+    elif isinstance(model_object, basestring):
```
Comment (quarckster) [inline cfme/tests/webui/test_splitter.py]: **REQUIRED**
Please use `six` for python3 compatibility:
```python
import six
...
elif isinstance(model_object, six.string_types):
```

File: cfme/tests/webui/test_splitter.py (hunk 1 above)
Comment (quarckster) [reply cfme/tests/webui/test_splitter.py]: **REQUIRED**
After adding `Exception` other exceptions don't make sense. You should be more specific. If the required exception doesn't exist, create your own and catch it here.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request closed without merging**